lib/transport/plugin.js

```diff
@@ -107,7 +167,10 @@ PluginTransport.prototype.call = function (device, type, message) {
         }
     });
 
-    return new Promise(function (resolve, reject) {
+
+    // preventing call and enumerate at the same time
+    // (TODO: Shouldn't we also care about 2 calls at the same time?)
```

Comment:
Calls are queued inside the plugin. Exclusive lock on `enumerate()` is a workaround for a bug in hidapi for Linux.